extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -424,7 +424,7 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       setExps[FLEXIBLE_AD_SLOTS_EXP] &&
```

Comment:
Nit:  Why do you need to check that setExps[FLEXIBLE_AD_SLOTS_EXP] is truthy?  If it weren't, then it would never equal the EXPERIMENT branch.

Reply:
Fixed.